.github/workflows/release.yaml

```diff
@@ -23,3 +23,21 @@ jobs:
         run: npm publish --workspaces --access public
         env:
           NODE_AUTH_TOKEN: ${{secrets.NPM_ACCESS_TOKEN}}
+
+  end-to-end:
```

Comment:
If I understand it correctly, a broken package will first make it to npm and then trigger a failure in the end-to-end test. Ideally this should be done before the release.
I looked a bit at the npm command and here is an alternative:
- Make a .tgz file for each package:
```
npm pack --workspace=@jazzer.js/instrumentor
npm pack --workspace=@jazzer.js/core
etc..
```
- Install the tar-files in the `end-to-end` directory: `npm i --save *.tgz` and run the test.

I wonder if this can work as a regular test, so that we see a failing test before running the release pipeline.